tobius/istruthy
# Update codebase

## Update codebase

> Update codebase (last updated was three years ago)

### Changes

- Use new `.editorconfig`
- Prefer Airbnb standards in `.eslintrc`
- Ignore `*.swp` files in .gitignore (Vim-friendly)
- Use MIT link instead of LICENSE file
- Add `.nvmrc` and use Node v16
- Update all devDependencies
- Use chai.assert instead of chai.expect
- Remove hard-coded eslint-disable rules
- Add `PULL_REQUEST_TEMPLATE`
- Fix lint errors
- Bump version revision from `0.2.6` to `0.2.7`

### Semantics

- [ ] This PR will introduce breaking changes (major)
- [ ] This PR will introduce new functionality (minor)
- [x] This PR will fix or tweak existing functionality (patch)

### Categories

- [ ] Configs
- [ ] Bugfixes
- [ ] File Structure
- [ ] Provisioning

### Validation Steps

1. Run `nvm use` to load Node v16
1. Run `npm run dev` (which runs `npm run test` && `npm run lint`)
1. See no errors

### Deployment Steps

1. Merge
